mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -37,10 +37,10 @@ custom_code:
   custom_import: 'templates/terraform/custom_import/monitoring_monitored_project.go.tmpl'
   test_check_destroy: 'templates/terraform/custom_check_destroy/monitoring_monitored_project.go.tmpl'
 error_retry_predicates:
-
   - 'transport_tpg.IsMonitoringPermissionError'
 schema_version: 1
 state_upgraders: true
+exclude_sweeper: true
```

Comment:
why is this excluded?

Reply:
It's done at the organization level and not the project level.

Reply:
interesting! Another potential case that would be fixed by moving to more complex list URL handling.